src/expandableCalendar/index.js

```diff
@@ -301,9 +295,13 @@ class ExpandableCalendar extends Component {
         toValue: this._height,
         speed: SPEED,
         bounciness: BOUNCINESS,
-        useNativeDriver: false
+        useNativeDriver: false,
       }).start(this.onAnimatedFinished);
 
+      if (this.props.onCalendarToggled) {
```

Comment:
Why not use _.invoke instead?

Reply:
Also, why not invoke it on 'onAnimatedFinished' instead?

Reply:
that's a simple callback, so I prefer to use it natively instead of lodash, that's easier to debug
could be changed also to `this.props.onCalendarToggled?.(isOpen)` wdyt?

I need to pass there position is it opened or closed and it counts directly at this place. So that's why it's called there.

Reply:
Yes. Better.